api/v1alpha1/flowcollector_types.go

```diff
@@ -262,6 +262,10 @@ type FlowCollectorFLP struct {
 	//+kubebuilder:default:=true
 	// EnableKubeProbes is a flag to enable or disable Kubernetes liveness/readiness probes
 	EnableKubeProbes bool `json:"enableKubeProbes,omitempty"`
+
+	//+kubebuilder:default:=true
+	// Set true to drop fields that are known to be unused by OVS, in order to save storage space.
+	DropUnusedFields bool `json:"dropUnusedFields,omitempty"`
```

Comment:
@mariomac I'm adding this field in FLP section as this is where it has an impact, however following the discussion on API review, maybe it makes more sense to have it under "ipfix"? (because it only has effect with ipfix, it would be ignored when agent is ebpf

Reply:
Ok, when this PR is merged, I'll rebase my branch and incorporate these changes

Reply:
Thinking it twice, I think this field could still be in the FLP section, as:
1. It's a configuration for FLP
2. Despite it's useless in EBPF, it could still have generic functionality if in the future we use other technologies/agents.